src/featbin/compute-kaldi-pitch-feats.cc

```diff
@@ -37,8 +37,8 @@ int main(int argc, char *argv[]) {
         "compute-kaldi-pitch-feats --sample-frequency=8000 scp:wav.scp ark:- \n"
         "\n"
         "See also: process-kaldi-pitch-feats, compute-and-process-kaldi-pitch-feats\n";
-    
```

Comment:
My editor shows blank spaces at the end of line and I removed the blank spaces automatically from lots of files.